src/pybind/mgr/crash/module.py

```diff
@@ -25,23 +103,23 @@ def handle_command(self, inbuf, command):
 
         return handler(self, command, inbuf)
 
-    @staticmethod
-    def validate_crash_metadata(inbuf):
-        # raise any exceptions to caller
-        metadata = json.loads(inbuf)
-        if 'crash_id' not in metadata:
-            raise AttributeError("missing 'crash_id' field")
-        return metadata
-
-    @staticmethod
-    def time_from_string(timestr):
+    def time_from_string(self, timestr):
         # drop the 'Z' timezone indication, it's always UTC
         timestr = timestr.rstrip('Z')
         try:
             return datetime.datetime.strptime(timestr, DATEFMT)
         except ValueError:
             return datetime.datetime.strptime(timestr, OLD_DATEFMT)
 
+    def validate_crash_metadata(self, inbuf):
+        # raise any exceptions to caller
+        metadata = json.loads(inbuf)
+        for f in ['crash_id', 'timestamp']:
+            if f not in metadata:
+                raise AttributeError("missing '%s' field" % (f))
```

Comment:
`f` doesn't need to be inside parenthesis. I think this line thought of the need for a tuple, but if that is the case a comma would be required. 
```suggestion
                raise AttributeError("missing '%s' field" % f)
```